localization/english/wc_general_l_english.yml

```diff
@@ -68,4 +68,8 @@
  DECLINE_OPT:3 "Decline."
  
  GREAT_OPT:3 "Great!"
- ALAS_OPT:3 "Alas!"
\ No newline at end of file
+ ALAS_OPT:3 "Alas!"
+
+ attempting_reunification_modifier:0 "Attempting Reunification"
+ attempting_reunification_modifier_desc:0 "[offer_vassalization|E] Acceptance: #P +25#!"
+ offer_vassalization_attempting_reunification_modifier_tt:0 "You are [GetModifier( 'attempting_reunification_modifier' ).GetNameWithTooltip|L]: $VALUE|=+0$"
```

Comment:
Move it to `wc_modifiers_l_english.yml`.

Reply:
Moved